plugin/src/arch/aarch64/debug.rs

```diff
@@ -116,53 +119,120 @@ pub fn format_opdata(name: &str, data: &Opdata) -> Vec<String> {
                 Matcher::Ident => write!(buf, "{}", arg_names[0]).unwrap(),
                 Matcher::Cond => write!(buf, "<cond>").unwrap(),
                 Matcher::Imm => write!(buf, "#{}", arg_names[0]).unwrap(),
-                Matcher::W =>   write!(buf, "W{}", arg_names[0]).unwrap(),
-                Matcher::X =>   write!(buf, "X{}", arg_names[0]).unwrap(),
+                Matcher::W => write!(buf, "W{}", arg_names[0]).unwrap(),
+                Matcher::X => write!(buf, "X{}", arg_names[0]).unwrap(),
                 Matcher::WSP => write!(buf, "W{}|WSP", arg_names[0]).unwrap(),
                 Matcher::XSP => write!(buf, "X{}|SP", arg_names[0]).unwrap(),
-                Matcher::B =>   write!(buf, "B{}", arg_names[0]).unwrap(),
-                Matcher::H =>   write!(buf, "H{}", arg_names[0]).unwrap(),
-                Matcher::S =>   write!(buf, "S{}", arg_names[0]).unwrap(),
-                Matcher::D =>   write!(buf, "D{}", arg_names[0]).unwrap(),
-                Matcher::Q =>   write!(buf, "Q{}", arg_names[0]).unwrap(),
+                Matcher::B => write!(buf, "B{}", arg_names[0]).unwrap(),
+                Matcher::H => write!(buf, "H{}", arg_names[0]).unwrap(),
+                Matcher::S => write!(buf, "S{}", arg_names[0]).unwrap(),
+                Matcher::D => write!(buf, "D{}", arg_names[0]).unwrap(),
+                Matcher::Q => write!(buf, "Q{}", arg_names[0]).unwrap(),
                 Matcher::V(s) => {
                     let width = if i == 0 { 16 } else { 8 };
-                    write!(buf, "V{}.{}{}", arg_names[0], size_to_string(*s), width / s.in_bytes()).unwrap();
-                },
-                Matcher::VStatic(s, c) => write!(buf, "V{}.{}{}", arg_names[0], size_to_string(*s), c).unwrap(),
-                Matcher::VElement(s) => write!(buf, "V{}.{}[{}]", arg_names[0], size_to_string(*s), arg_names[1]).unwrap(),
-                Matcher::VElementStatic(s, element) => write!(buf, "V{}.{}[{}]", arg_names[0], size_to_string(*s), element).unwrap(),
-                Matcher::VStaticElement(s, c) => write!(buf, "V{}.{}{}[{}]", arg_names[0], size_to_string(*s), c, arg_names[1]).unwrap(),
+                    write!(
```

Comment:
These ones I'm a bit on edge about. I think they're being triggered by a column width check? Looks like it rejects over 100 characters. Personally I'm fine with 120 at least, although the last does go over that so maybe just exempting this section is for the best.